### ISSUE TITLE
[Bug]: polybar disable IPC channels

### ISSUE THREAD

**Issue by numberokta** [polybar/polybar#2683]
### Checklist

- [X] I have read the appropriate section in the [contributing guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
- [X] I believe this issue is a problem with polybar itself and not a misconfiguration on my part
- [X] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that may have already reported this problem
- [X] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.
- [X] I have followed the [debugging guide](https://github.com/polybar/polybar/wiki/Debugging-your-Config) to narrow down the problem to a minimal config.

### Steps to reproduce

polybar -c ~/polybarrc -q main

### Minimal config

```dosini
[global/wm]
margin-bottom = 0
margin-top = 0
include-file = $POLYBARDIR/link
[bar/main]
monitor =
monitor-fallback =
monitor-strict = false
override-redirect = true
fixed-center = true
width = 100%:-20
height = 30
offset-x = 10
offset-y = 10
background = ${color.alpha}
foreground = ${color.foreground}
radius-top = 0.0
radius-bottom = 0.0
underline-size = 2
underline-color = ${color.foreground}
border-size = 0
border-color = ${color.background}
padding = 0
module-margin-left = 0
module-margin-right = 0
font-0 = "DejaVu Sans Mono:size=10;3"
include-file = $POLYBARDIR/sort
separator =
dim-value = 1.0
wm-name = 
locale = 
tray-position = right
tray-detached = false
tray-maxsize = 30
tray-offset-x = 3
tray-offset-y = 0
tray-padding = 3
tray-scale = 1.0
wm-restack = bspwm
enable-ipc = true
click-left = 
click-middle = 
click-right =
scroll-up =
scroll-down =
double-click-left =
double-click-middle =
double-click-right =
cursor-click = pointer
cursor-scroll = 
[settings]
throttle-output = 5
throttle-output-for = 10
screenchange-reload = false
compositing-background = source
compositing-foreground = over
compositing-overline = over
compositing-underline = over
compositing-border = over
pseudo-transparency = true
```

### Polybar log

```text
error: Disabling IPC channels due to error: /home/test/polybar/polybar-3.6.2/src/components/eventloop.cpp:112: libuv error for 'uv_pipe_bind(get(), path.c_str())': permission denied
```

### Expected behavior

i use **enable-ipc = true** in my config file and i expect **IPC** channels work, but polybar disable it.

### Actual behavior

polybar crashed with error on starting

### Window Manager and Version

bspwm 0.9.10

### Linux Distribution

ubuntu 21.10

### Polybar version

```text
polybar 3.6.2
```

### Additional Context / Screenshots

i test this in version 3.6.0 and later and **just sometimes** (not always) that error happen.
when i logout and login with another user and run polybar with same config that error happen.
i think that is becuse, _Starting with version 3.6.0, the underlying IPC mechanism has been completely changed_.
 in version 3.6.0 and later when polybar starts, it makes directory **/tmp/polybar** for **IPC**.
when i logout and login with other user, that directory is still there with other owner and polybar do not have permission to create IPC file in there.
when i remove **/tmp/polybar** whith root access and then run polybar it works perfectly.

**Comment by patrick96** [comment]
Oh dang, you're right, that doesn't work for multiple users. Should be easy enough to fix, we just have to use a different folder per use (`/tmp/polybar-UID`).

This will be fixed in 3.6.3

**Comment by numberokta** [comment]
Thanks for your fast response.
awesome app by the way.
